game_event/tests.py

```diff
@@ -75,3 +82,32 @@ def test_create_game_event_with_null_ball_game(self, saved_court):
             GameEvent.objects.create(id=TEST_ID, time=TEST_TIME, level_of_game=TEST_LEVEL,
                                      min_number_of_players=TEST_MIN, max_number_of_players=TEST_MAX,
                                      court=saved_court,).full_clean()
+
+    def test_create_game_event_valid_form(self, client, saved_court, court_ball_game):
+        url = reverse(create_game_event)
+        form_data = {
+            'time': "2034-05-10T14:30",
+            'level_of_game': 5,
+            'min_number_of_players': 2,
+            'max_number_of_players': 4,
+            'court': saved_court.courtID,
+            'ball_game': TEST_BALL_GAME,
+        }
+        client.post(url, data=form_data)
+        assert GameEvent.objects.count() == 1
```

Comment:
This test will fail when you have data. To prevent this, consider adding the previous size of the data as a variable before running the test. Then, you can test that the new size is equal to the previous size plus one

Reply:
Fixed